code/game/objects/items/weapons/RCD.dm

```diff
@@ -11,312 +11,180 @@ RCD
 	icon_state = "rcd"
 	opacity = 0
 	density = 0
-	anchored = 0
+	anchored = 0.0
```

Comment:
No we don't, but coders in the past wrote the 0 as 0.0, it's still the same to the game when it's checking for it.